main.c

```diff
@@ -218,6 +218,8 @@ bool default_filename(char *filename, size_t n, int filetype) {
 #else
 		assert(false);
 #endif
+	default:
+		ext = "png";
```

Comment:
I'd prefer not to add a `default`, and instead to initialize `ext` with NULL and assert it's not null after the switch. That way, adding a new file type will trigger a warning and let us know that this switch needs updating.

Reply:
Sounds like a better approach. Is this what you had in mind?